### PR TITLE
gha: aks: Use short SHA in cluster name

### PR DESCRIPTION
**Pull request by jepio**
Full SHA is 40 characters, while AKS cluster name has a limit of 63. Trim the SHA to 12 characters, which is widely considered to be unique enough and is short enough to be used in the cluster name

Fixes: #7010

### CODE REVIEW

File: .github/workflows/run-k8s-tests-on-aks.yaml [hunk 1]
```diff
@@ -46,11 +46,16 @@ jobs:
             -p "${{ secrets.AZ_PASSWORD }}" \
             --tenant "${{ secrets.AZ_TENANT_ID }}"
 
+      - name: Format cluster name
+        run: |
+          rev=$(echo "${{ github.event.pull_request.head.sha }}" | cut -c1-12)
```
Comment (fidencio) [inline .github/workflows/run-k8s-tests-on-aks.yaml:51]:
```suggestion
          rev=$(git rev-parse --short HEAD)
```

We can do that as at this point the repo has been pulled and we're at the `github.event.pull_request.head.sha` commit.

**Review by fidencio — COMMENTED** [review]
@jepio, one suggestion :-)

**Review by fidencio — COMMENTED** [review]
@jepio, one suggestion :-)

**Review by fidencio — APPROVED** [review]
lgtm, thanks @jepio!

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks @jepio

**Pull request merged**